ingestion-beam/src/main/java/com/mozilla/telemetry/util/JsonValidator.java

```diff
@@ -21,4 +27,18 @@ public void validate(Schema schema, ObjectNode json) throws JsonProcessingExcept
     validator.performValidation(schema, Json.convertValue(json, JSONObject.class));
   }
 
+  /**
+   * Validate a Jackson array.
+   *
+   * <p>Note this requires serializing the array back to a JSON string because
+   * the Jackson mapper cannot convert an ArrayNode to an org.json.JSONArray
+   * type.
+   */
+  public void validate(Schema schema, ArrayNode json) throws JsonProcessingException, IOException {
+    ObjectMapper mapper = new ObjectMapper();
+    mapper.registerModule(new JsonOrgModule());
+    JsonNode root = mapper.valueToTree(json);
+    JSONArray jsonArray = mapper.treeToValue(root, JSONArray.class);
+    validator.performValidation(schema, jsonArray);
+  }
```

Comment:
See https://github.com/mozilla/gcp-ingestion/pull/1260 for an implementation of `validate` for ArrayNode.